pkg/staticnetworkconfig/generator.go

```diff
@@ -13,39 +14,49 @@ import (
 	"github.com/openshift/assisted-service/pkg/executer"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"golang.org/x/sync/semaphore"
 	"gopkg.in/ini.v1"
 	"gopkg.in/yaml.v2"
 )
 
 const staticNetworkConfigHostsDelimeter = "ZZZZZ"
 const hostStaticNetworkDelimeter = "HHHHH"
 
+type Config struct {
+	MaxConcurrentGenerations int `envconfig:"MAX_CONCURRENT_NMSTATECTL_GENERATIONS" default:"30"`
```

Comment:
This can just be an int64 to avoid the type conversion later.

Reply:
fixed